crates/ra_assists/Cargo.toml

```diff
@@ -22,4 +22,5 @@ ra_prof = { path = "../ra_prof" }
 ra_db = { path = "../ra_db" }
 ra_ide_db = { path = "../ra_ide_db" }
 hir = { path = "../ra_hir", package = "ra_hir" }
+hir_expand = { path = "../ra_hir_expand", package = "ra_hir_expand" }
```

Comment:
I think the idea is to export everything from `ra_hir` and only depend on that crate.

Reply:
Thanks for pointing out. Already fixed!